app/components/BasicCatchBoundary.tsx

```diff
@@ -8,9 +8,9 @@ export default function BasicCatchBoundary({
   return (
     <div className="mx-auto min-h-[50vh] w-screen max-w-3xl px-4 pt-32 pb-12">
       <div className="prose prose-lg prose-slate lg:prose-xl">
-        <h2>Oh no! A {status} error!</h2>
-        {status === 404 && <p>We couldn't find that page!</p>}
-        <p>This is what the machines tell us:</p>
+        <h2>Oh no! A {status} error! 🙀</h2>
+        {status === 404 && <p>We couldn't find that page! 😬</p>}
+        <p>This is what the machines tell us 🤖</p>
         <pre className="bg-slate-700">
           <code>{statusText}</code>
```

Comment:
<img width="843" alt="Screenshot 2022-09-02 at 09 18 10" src="https://user-images.githubusercontent.com/88032788/188080757-0b74afef-670c-44d1-883b-585214391845.png">

Should we display the `This is what the machine tell us....` only if we have statusText? Otherwise the text looks a bit broken. Maybe just display a default "Try again later" if we don't have statusText

Reply:
Oh, yes! Will change that.